pages/pipelines/environment_variables.md.erb

```diff
@@ -62,14 +98,42 @@ The following environment variables are automatically provided to every job, and
     <th><code>BUILDKITE_COMMAND</code></th>
     <td>The command that will be run for the job. <p class="Docs__api-param-eg"><em>Example:</em> <code>"script/buildkite/specs"</code></p></td>
   </tr>
+  <tr>
+    <th><code>BUILDKITE_COMMAND_EVAL</code></th>
+    <td>A boolean to allow or deny running arbitrary commands on the agent. <p class="Docs__api-param-eg"><em>Default:</em> <code>"true"</code></p></td>
+  </tr>
   <tr>
     <th><code>BUILDKITE_COMMIT</code></th>
     <td>The commit of the build. <p class="Docs__api-param-eg"><em>Example:</em> <code>"83a20ec058e2fb00e7fa4558c4c6e81e2dcf253d"</code></p></td>
   </tr>
+  <tr>
+    <th><code>BUILDKITE_CONFIG_PATH</code></th>
+    <td> <p class="Docs__api-param-eg"><em>Default:</em> <code>"/buildkite/buildkite-agent.cfg"</code></p></td>
+  </tr>
   <tr>
     <th><code>BUILDKITE_DISABLE_GIT_SUBMODULES</code></th>
     <td>When set to <code>true</code> prevents fetching of any git submodules during checkout. This variable is used by the <code>buildkite-agent</code> bootstrap script. <p class="Docs__api-param-eg"><em>Example:</em> <code>"true"</code></p></td>
   </tr>
+  <tr>
+    <th><code>BUILDKITE_ENV_FILE</code></th>
+    <td> <p class="Docs__api-param-eg"><em>Example:</em> <code>"/tmp/job-env-36711a2a-711a-484e-b180-e1b3711a80cf51b18711a"</code></p></td>
```

Comment:
Do you want it in the list if it's internal? Is it something that people will see and wonder about?

Reply:
I just did some digging, and it's not actually internal at all… it was added to help people wanting to pass envs into Docker: https://github.com/buildkite/agent/pull/643 https://github.com/buildkite/agent/pull/682

So let's document it!

```suggestion
    <td>The path to the file containing the job’s environment variables. <p class="Docs__api-param-eg"><em>Example:</em> <code>"/tmp/job-env-36711a2a-711a-484e-b180-e1b3711a80cf51b18711a"</code></p></td>
```

We should probably document this feature, and link to it from this description though, because I don't think people will understand what this is from this description 🤔 At least it's a start I suppose, and they can go digging.